tests/test_regr.py

```diff
@@ -118,6 +118,9 @@ def test_check_package___init__(finalize_linter: PyLinter) -> None:
     assert checked == ["__init__"]
 
 
+# pylint: disable-next=fixme
+# TODO: Optparse: Test are broken because of property shenanigans of config attribute
+@pytest.mark.xfail
```

Comment:
Why not update the test ? We changed the Pylinter code, it's logical that the result of the test change ?

Reply:
Yeah, because of the three-step property chaining `astroid` can't infer `linter.config` anymore and returns `Uninferable`. When we simplify those attributes this will probably suddenly no longer fail 🎉

Reply:
All right !